service/history/replication/raw_task_converter.go

```diff
@@ -193,6 +193,9 @@ func convertActivityStateReplicationTask(
 						LastStartedRedirectCounter: activityInfo.GetUseWorkflowBuildIdInfo().GetLastRedirectCounter(),
 						BaseExecutionInfo:          persistence.CopyBaseWorkflowInfo(mutableState.GetBaseWorkflowInfo()),
 						VersionHistory:             versionhistory.CopyVersionHistory(currentVersionHistory),
+						FirstScheduledTime:         activityInfo.FirstScheduledTime,
+						LastAttemptCompleteTime:    activityInfo.LastAttemptCompleteTime,
+						Stamp:                      activityInfo.Stamp,
```

Comment:
Also need to update standby cluster logic to apply those fields (executable_activity_state_task.go, activity_state_replicator.go etc.) Can probably do all replication related logic in a separate PR.

Reply:
I updated SyncActivityRequest, ActivitySyncInfo, etc. May be even more then needed (information in SyncActivityRequest is duplicated in ActivitySyncInfo, but also ActivitySyncInfo is created then from SyncActivityRequest). Please check if all code path are covered.

Reply:
I think I update everything .